sunbeam-python/sunbeam/versions.py

```diff
@@ -8,6 +8,7 @@
 JUJU_CHANNEL = "3.6/stable"
 JUJU_BASE = "ubuntu@24.04"
 OPENSTACK_CHANNEL = "2024.1/stable"
+MANILA_CHANNEL = "2025.1/edge"
```

Comment:
Can we remove this since manifests/*.yml are already updated?

Reply:
Done